src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -18,22 +18,9 @@
 
     public static final String MESSAGE_SUCCESS = "New task added: %1$s";
 
-    private final Task toAdd;
-
-    /**
-     * Convenience constructor using raw values.
-     *
-     * @throws IllegalValueException if any of the raw values are invalid
-     */
-    public AddCommand(String name) throws IllegalValueException {
-        this.toAdd = new Task(new Name(name));
-    }
-
-    @Override
-    public CommandResult execute() {
-        assert model != null;
-        model.addTask(toAdd);
-        return new CommandResult(String.format(MESSAGE_SUCCESS, toAdd));
-    }
-
+	@Override
+	public CommandResult execute() {
+		// TODO Auto-generated method stub
+		return null;
+	}
```

Comment:
But Parser depends on AddCommand.COMMAND_WORD, which is add. I have deleted the AddCommand file and replaced AddCommand.COMMAND_WORD in Parser with "add", though i am unsure if this will violate code quality